### PR TITLE
fix(swipe-carousel): render cards without images and skip empty rows

### PR DESCRIPTION
**Pull request by amarghioali**
- Simplify card detection to check cell count instead of content
- Add empty row check for single-cell rows to clean up markdown
- Remove placeholder image reference that doesn't exist in DAM
- Cards now render in UE even when images are missing

## Issue

Fixes #I4

- Before: https://main--extweb-academy--aemsites.aem.page/
- After: https://4-block-carousel--extweb-academy--aemsites.aem.page/library/carousel

### CODE REVIEW

File: blocks/swipe-carousel/swipe-carousel.js [hunk 1]
```diff
@@ -15,31 +15,56 @@ export default function decorate(block) {
   [...block.children].forEach((row) => {
     const cells = row.querySelectorAll(':scope > div');
 
-    // Detect if this row is a link-only row (for linkText/link extraction)
+    // Check if row is empty (UE creates empty rows for unused block-level fields)
+    const isEmpty = cells.length > 0
+      && Array.from(cells).every((cell) => !cell.textContent.trim()
+        && !cell.querySelector('picture, img, a, h1, h2, h3, h4, h5, h6'));
```
[the remark this answers is not in the document]
Comment (amarghioali) [reply blocks/swipe-carousel/swipe-carousel.js]: Thank you. I've updated the logic to ensure the isEmpty check now correctly handles all HTML elements, including ul, code blocks, and other elements that weren't covered by the initial selector list, preventing potential edge case misses.

I've also resolved all ESLint errors and now I will merge the branch so we can fully test the functionality in the UE.

Everything should be fine now.
